truss/templates/control/control/endpoints.py

```diff
@@ -18,13 +22,17 @@ def index():
     return {}
 
 
-@control_app.get("/v1/{full_path:path}")
-@control_app.post("/v1/{full_path:path}")
-async def proxy(full_path: str, request: Request):
+async def proxy(request: Request):
     inference_server_port = request.app.state.inference_server_port
     inference_server_process_controller = (
         request.app.state.inference_server_process_controller
     )
+    inference_server_port = request.app.state.inference_server_port
+    client = httpx.AsyncClient(base_url=f"http://localhost:{inference_server_port}")
```

Comment:
Is there a session/pool that can be used? Does this setup a new connection each time?

Reply:
`0.0.0.0` or `localhost`?

Reply:
```
with httpx.Client(base_url="http://...") as client:
   ....
```
will handle the gc/close more explicitly.

Reply:
```
limits = httpx.Limits(max_keepalive_connections=5, max_connections=10)
client = httpx.Client(limits=limits)
```

connection pool is probably going to be more performant.

Reply:
There is a session that I can set up. Let me do that and test it.